### PR TITLE
staging <- dev

### PR DESCRIPTION
**Pull request by rtirserio**

### CODE REVIEW

File: talentmap_api/fsbid/services/bid_season.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 def get_bid_seasons(bsn_future_vacancy_ind, jwt_token):
     # set future vacancy indicator - default to 'Y'
     future_vacancy_ind = bsn_future_vacancy_ind if bsn_future_vacancy_ind else 'Y'
-    url = f"{API_ROOT}/bidSeasons?bsn_future_vacancy_ind={future_vacancy_ind}"
```
Comment (burgwyn) [inline talentmap_api/fsbid/services/bid_season.py]: I was testing on site today and I saw this field as `bsn_future_vacancy_ind`

Comment (rtirserio) [reply talentmap_api/fsbid/services/bid_season.py]: its getting sent to the fsbid as `bsn_future_vacancy_ind` just named something different on the local variable

Comment (burgwyn) [reply talentmap_api/fsbid/services/bid_season.py]: Return value is `bsn_future_vacancy_ind` from the FSBid API.  I don't see this field as required anymore.  Unfortunately the endpoint is down for me to test.

Comment (burgwyn) [reply talentmap_api/fsbid/services/bid_season.py]: Disregard, I was looking at and old endpoint

**Pull request merged**